python/python/bystro/covariance/_covariance_np.py

```diff
@@ -0,0 +1,96 @@
+"""
+This implements two objects, the empirical covariance estimator used as a 
+baseline comparison method. It also implements BayesianCovariance, which
+implements MAP estimation using several common priors.
+
+Objects
+-------
+EmpiricalCovariance(BaseCovariance)
+    
+
+"""
+import numpy as np
+from ._base_covariance import BaseCovariance
```

Comment:
I believe the house style is to use absolute imports, i.e. `from bystro.covariance._base_covariants import BaseCovariance`

Reply:
OK will change